core/common/io/src/main/java/org/eclipse/rdf4j/common/io/package-info.java

```diff
@@ -0,0 +1,8 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eclipse RDF4J contributors. All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Distribution License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/org/documents/edl-v10.php.
```

Comment:
It looks like the copyright header was accidentally malformed in several places - it's missing newlines.

It should be like this:

```
/*******************************************************************************
 * Copyright (c) 2022 Eclipse RDF4J contributors.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Distribution License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/org/documents/edl-v10.php.
 *******************************************************************************/
```

However, given that these are `package-info.java` files, I would be fine with just removing these copyright headers. @hmottestad I remember something being said about copyright headers in the last Eclipse review, were package-info.java files included in that comment?

Reply:
That comment was about some test files. I think we should stick with copyright headers in all *.java files.

Reply:
It is very useful for us because we could now use rdf4j in our platform. The Platform is OSGi based and so we couldn't use RDF4J without the proper Manifest-information. Also if you want to use it in eclipse plugins you would need it because this is also osgi based.